components/motor/roboclaw/roboclaw.go

```diff
@@ -4,21 +4,27 @@ package roboclaw
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"math"
 	"time"
 
 	"github.com/CPRT/roboclaw"
 	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
```

Comment:
[opt] I think the "errors" package is preferred. IS there a reason it has switched? I don't know if this one is as maintained.

Reply:
I switched it so I could use errors.Wrap, and to be consistent with the other motor drivers, almost all of them use this one.